src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    protected LocalDate by;
+
+    public Deadlines(String description, LocalDate by) {
+        super(description);
+        this.by = by;
+    }
+
+    public static Deadlines load(String taskDetails) {
+        String[] splitTaskDetails = taskDetails.split("\\|",4);
```

Comment:
Small issue, missing a space after the `,` here.

(there's other instances of this from the same code in the other classes)